agb/src/display/affine.rs

```diff
@@ -193,25 +269,25 @@ impl Mul for AffineMatrix {
     }
 }
 
-impl Mul<Num<i32, 8>> for AffineMatrix {
-    type Output = Self;
-    fn mul(self, rhs: Num<i32, 8>) -> Self::Output {
-        self * AffineMatrix {
-            a: rhs,
-            b: 0.into(),
-            c: 0.into(),
-            d: rhs,
-            x: 0.into(),
-            y: 0.into(),
-        }
-    }
-}
+// impl Mul<Num<i32, 8>> for AffineMatrix {
+//     type Output = Self;
+//     fn mul(self, rhs: Num<i32, 8>) -> Self::Output {
+//         self * AffineMatrix {
+//             a: rhs,
+//             b: 0.into(),
+//             c: 0.into(),
+//             d: rhs,
+//             x: 0.into(),
+//             y: 0.into(),
+//         }
+//     }
+// }
```

Comment:
I guess scalar multiplication doesn't make too much sense with the translation?

Reply:
mind deleting the commented out code?

Reply:
Yeah, the issue is that I don't keep track of the lower right entry and treat it as being 1